charts/prometheus/templates/network-policy.yaml

```diff
@@ -12,5 +12,5 @@ spec:
       {{- include "prometheus.server.matchLabels" . | nindent 6 }}
   ingress:
     - ports:
-      - port: 9090
+      - port: {{ include "prometheus.port" . }}
```

Comment:
does this take port name? unsure so keeping as is for now

Reply:
Yes, it does.

```suggestion
      - port: {{ default (include "prometheus.port" .) .Values.server.portName }}
```